seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -374,8 +374,9 @@
         "</div>"
       ],
       "tests": [
+        "assert($('p').children('strong').length == 1, 'message: The <code>p</code> tag should wrap around and contain the <code>strong</code> tag.');",
```

Comment:
Would this message be better as "The `<strong>` tag should be inside the `<p>` tag"?